pkg/scheduler/metrics/metrics.go

```diff
@@ -164,6 +164,14 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		}, []string{"type"})
 
+	unschedulableReasons = metrics.NewGaugeVec(
```

Comment:
CounterVec

Reply:
Aldo, see discussion on the main issue: https://github.com/kubernetes/kubernetes/issues/102867

Reply:
Ah ok, so it's more like pending_pods metric, rather than schedule_attempts. Sounds good.

Reply:
Correct, we want to track the current actual number of unschedulable pods.